exasol/bucketfs/_buckets.py

```diff
@@ -119,6 +128,7 @@ def __init__(
         username: str,
         password: str,
         verify: bool | str = True,
+        service_name: Optional[str] = None
```

Comment:
why add the service name this kinda breaks the layers?
(Lower layer now knows the upper layer)